### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by brim-borium**

### CODE REVIEW

**Review by Joran-Dob — APPROVED** [review]
Great work @brim-borium 🎊

**Comment by fotiDim** [comment]
Are we sure that 🚧 is visible on pub.dev? I am asking because `:construction_worker:` isn't.

**Comment by brim-borium** [comment]
> Are we sure that 🚧 is visible on pub.dev? I am asking because `:construction_worker:` isn't.

I tried to use the native emojis that are visible on other packages on pub.dev so I hope that this is visible

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 1.0.0
+* adds support for iOS (thanks [fotiDim](https://github.com/fotiDim))
+* adapts to breaking changes in the spotify android sdk (thanks [itsMatoosh](https://github.com/itsMatoosh))
+* extends usage of the spotify android sdk functionalities (thanks [Joran-Dob](https://github.com/Joran-Dob))
```
Comment (fotiDim) [inline CHANGELOG.md:4]:
```suggestion
* Adds the `setShuffle()` and `setRepeatMode()` APIs for android (thanks [Joran-Dob](https://github.com/Joran-Dob))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+* renames logout to disconnect
+* some minor bug fixing
```
Comment (fotiDim) [inline CHANGELOG.md:6]:
```suggestion
* Adds `accessToken` as an optional parameter to `connectToSpotifyRemote()` (only supported on iOS)
* some minor bug fixing
```

**Comment by fotiDim** [comment]
2 suggestions and 1 commit that you can review @brim-borium

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,12 @@
+## 1.0.0
+* __adds support for iOS__ 🎉 (thanks [fotiDim](https://github.com/fotiDim))
+* adapts to breaking changes in the spotify android sdk (thanks [itsMatoosh](https://github.com/itsMatoosh))
+* implemented PKCE auth flow for the web implementation (thanks [itsMatoosh](https://github.com/itsMatoosh))
+* adds the `setShuffle()` and `setRepeatMode()` APIs for android (thanks [Joran-Dob](https://github.com/Joran-Dob))
```
Comment (fotiDim) [inline CHANGELOG.md:5]:
```suggestion
* adds the `setShuffle()` and `setRepeatMode()` APIs for Android (thanks [Joran-Dob](https://github.com/Joran-Dob))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,12 @@
+## 1.0.0
+* __adds support for iOS__ 🎉 (thanks [fotiDim](https://github.com/fotiDim))
+* adapts to breaking changes in the spotify android sdk (thanks [itsMatoosh](https://github.com/itsMatoosh))
+* implemented PKCE auth flow for the web implementation (thanks [itsMatoosh](https://github.com/itsMatoosh))
+* adds the `setShuffle()` and `setRepeatMode()` APIs for Android (thanks [Joran-Dob](https://github.com/Joran-Dob))
+* renames logout to disconnect
+* Adds `accessToken` as an optional parameter to `connectToSpotifyRemote()` (only supported on iOS)
```
Comment (fotiDim) [inline CHANGELOG.md:7]:
```suggestion
* adds `accessToken` as an optional parameter to `connectToSpotifyRemote()` (only supported on iOS)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,12 @@
+## 1.0.0
+* __adds support for iOS__ 🎉 (thanks [fotiDim](https://github.com/fotiDim))
+* adapts to breaking changes in the spotify android sdk (thanks [itsMatoosh](https://github.com/itsMatoosh))
+* implemented PKCE auth flow for the web implementation (thanks [itsMatoosh](https://github.com/itsMatoosh))
```
Comment (fotiDim) [inline CHANGELOG.md:4]:
```suggestion
* implements PKCE auth flow for the web implementation (thanks [itsMatoosh](https://github.com/itsMatoosh))
```

**Review by fotiDim — APPROVED** [review]
I was missing a commit from before. Not sure what happened. I re-pushed it.

**Review by Joran-Dob — APPROVED** [review]

**Pull request merged**